taxonomies/additives.txt

```diff
@@ -4532,7 +4534,7 @@ e_number:en:270
 wikidata:en:Q161249
 organic_eu:en:authorized
 vegan:en:maybe
-vegetarian:en:yes
+vegetarian:en:maybe
```

Comment:
What is the issue here?

Reply:
From the UK Veg Society: "E270 Lactic acid
Obtained commercially by heating and fermenting carbohydrates such as sucrose, molasses, starch, or whey. Occurs naturally in blood, so could conceivably be of animal origin. Often used as a pH regulator in dairy products and drinks. Found in soft sweets, pickled vegetables, low-fat mayonnaise and sour dough bread."

Reply:
foodinfo: lactic acid and lactates can be consumed by all religious groups, vegans and vegetarians. Although the name refers to milk, it is mot made from milk and thus suitable for people with milk allergy or lactose intolerance.